ui/src/cohortReview/plugins/occurrenceTabs.tsx

```diff
@@ -0,0 +1,74 @@
+import { useCohortReviewContext } from "cohortReview/cohortReviewContext";
+import {
+  CohortReviewPlugin,
+  getCohortReviewPlugin,
+  registerCohortReviewPlugin,
+} from "cohortReview/pluginRegistry";
+import { useReviewSearchState } from "cohortReview/reviewHooks";
+import { GridBox } from "layout/gridBox";
+import { useMemo } from "react";
+import { CohortReviewPageConfig } from "underlaysSlice";
+import { Tabs } from "components/tabs";
+
+interface Config {
+  entity: string;
+  tabs: CohortReviewPageConfig[];
+}
+
+@registerCohortReviewPlugin("entityTabs")
```

Comment:
@tjennison-work The main reason I have this as a draft is not sure if this should be a core plugin or VUMC-specific.

Reply:
Sorry, I was out the past two days. Core sounds good. It's generically useful.